controller.go

```diff
@@ -106,14 +119,16 @@ type driverData struct {
 type driverTable map[string]*driverData
 type networkTable map[types.UUID]*network
 type endpointTable map[types.UUID]*endpoint
-type sandboxTable map[string]*sandboxData
+type containerTable map[string]*sandboxData
+type sandboxTable map[string]*sandbox.Sandbox
 
 type controller struct {
-	networks  networkTable
-	drivers   driverTable
-	sandboxes sandboxTable
-	cfg       *config.Config
-	store     datastore.DataStore
+	networks   networkTable
+	drivers    driverTable
+	sandboxes  sandboxTable
+	containers containerTable
```

Comment:
Lets simplify this through consolidation. Also, since sandboxData implements Sandbox interface, it is more appropriate to have a single collection named sandboxTable that carries just the sandboxData and eliminate containerTable altogether.